app/ts/LiquidityProvidingUI/Liquidity.tsx

```diff
@@ -215,47 +215,57 @@ export const Liquidity = ({ maybeReadClient, maybeWriteClient, universe, reputat
 	const clear = () => {
 		marketData.deepValue = undefined
 	}
+	useSignalEffect(() => {
+		selectedMarket.deepValue
+		clear()
+	})
```

Comment:
It may be better to just make the `refresh` function do `marketData.deepValue = undefined` at the start, rather than manually clearing via a signal effect.  Anytime refresh is called (for any reason) it would make sense to clear the market data and repopulate it as that function finishes, and having this separate `clear` function makes it easy to forget to clear before refreshing.